1.7/usage/tutorials/arangodb/index.md

```diff
@@ -46,45 +46,39 @@ Click the ArangoDB task to reveal the subtask that the framework has started:
 
 ![Tasks](img/tasks.png)
 
-Click “Open Service” to open the ArangoDB dashboard:
+Click “Open Service” to open the ArangoDB Webinterface.
 
-![Dashboard](img/dashboard.png)
-
-By default, ArangoDB will not expose itself to the outside. The IPs listed here are the internal IPs in the cluster. To access the nodes from the outside, we recommend using [sshuttle](https://github.com/sshuttle/sshuttle).
+![Dashboard](img/arangodb.png)
 
-The exact way to dig a tunnel using sshuttle varies from infrastructure to infrastructure. The following is an example for an AWS cluster:
+Congratulations! You now have ArangoDB running on DC/OS.
 
-```bash
-$ sshuttle --python /opt/mesosphere/bin/python3.4 -r core@54.171.143.132 10.0.0.0/8
-```
+## Using ArangoDB within the DC/OS cluster
 
-The IP may be extracted from the top left corner of the DC/OS web interface:
+Now that you have ArangoDB running you will likely want to fill it with some data and be able to use it as a data store for your applications.
 
-![Dashboard](img/ip.png)
+To do so you need to talk to the so called "Coordinators" of your ArangoDB cluster. As you can't be sure where they are located and due to the fact that they might move or change anytime throughout your cluster lifetime (task might fail, up- and downscaling and so forth) it is a bad idea to hardcode their IP addresses and ports in your applications.
 
-**Note:** Some sshuttle versions had problems during our tests. Version 0.77.3 worked properly for us.
+To connect to ArangoDB from the inside deploy the [ArangoDB Mesos HAProxy](https://github.com/arangodb/arangodb-mesos-haproxy).
 
-Afterwards, you should be able to access the internal IPs from outside. Click the coordinator link in the ArangoDB web interface to open the ArangoDB coordinator:
+Clone the repository and start the proxy:
 
-![Dashboard](img/arangodb.png)
+```bash
+$ git clone https://github.com/arangodb/arangodb-mesos-haproxy
+$ cd arangodb-mesos-haproxy
+$ dcos marathon app add marathon.json
+```
 
-Congratulations! You now have ArangoDB running on DC/OS.
+This will create a proxy within DC/OS which is always accessible via `arangodb-proxy.marathon.mesos:8529`. It will watch out for any coordinator changes and serve as a connection broker to the coordinators. Use this as the endpoint within your applications. To make it highly available and/or scale it you can simply add a few more instances via the marathon ui.
```

Comment:
Consider revising to: `To make your proxy highly available or scale it, you can simply add a few more instances via the Marathon UI.`